README.md

```diff
@@ -10,12 +10,17 @@ this will give you a virtual machine already configured and ready to start devel
 ## Installing manually
 
 If you prefer not to use the Docker, the following commands will get you started:
-commands can be converted to:
 
     $ carton install
+    $ cpanm -l local WWW::Form::UrlEncoded::XS  # Optional but recommended
```

Comment:
Where is this module used?  Rather than adding an extra install step we could just add it to the `cpanfile`.

Reply:
I got a warning when running front-end tests that something was falling back to PP since the XS was missing.  If it's OK to require an XS module for tests, I'll gladly drop it in the cpanfile!